docs/tutorial/tutorial_constellation.rst

```diff
@@ -47,3 +47,94 @@ The :py:attr:`~pysat._instrument.Instrument.name` values are: :py:data:`epam`,
 :py:data:`mag`, :py:data:`sis`, and :py:data:`swepam`.  The only
 :py:class:`~pysat._instrument.Instrument` defining attributes that are not
 unique are the :py:attr:`~pysat._instrument.Instrument.name` values.
+
+Use a Constellation sub-module
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Some pysatEcosystem packages, such as :py:mod:`pysatNASA`, contain sub-modules
+that specify commonly-used Constellations. This example uses the
+`ICON Instruments <https://pysatnasa.readthedocs.io/en/latest/supported_constellations.html#icon>`_
+to create a Constellation of ICON EUV, FUV, IVM, and MIGHTI data (excluding the
+line-of-sight winds).
+
+.. code:: python
+
+    import pysat
+    import pysatNASA
+
+    # Initalize the ICON Constellation using the ICON module
+    icon = pysat.Constellation(cont_module=pysatNASA.constellations.icon)
```

Comment:
```suggestion
    icon = pysat.Constellation(const_module=pysatNASA.constellations.icon)
```

Reply:
As currently defined at pysatNASA, the icon constellation will never load.  See https://github.com/pysat/pysatNASA/issues/152